framework/core/command_buffer.h

```diff
@@ -219,6 +220,14 @@ class CommandBuffer
 
 	void set_update_after_bind(bool update_after_bind_);
 
+	void reset_query_pool(const QueryPool &query_pool, uint32_t first_query, uint32_t query_count);
+
+	void begin_query(const QueryPool &query_pool, uint32_t query, VkQueryControlFlags flags);
+
+	void end_query(const QueryPool &query_pool, uint32_t query);
+
+	void write_timestamp(VkPipelineStageFlagBits pipelineStage, const QueryPool &query_pool, uint32_t query);
```

Comment:
snake_case

Reply:
It's hard switching coding styles sometimes ;-)
Done